src/components/ProgressBar.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+const container = {
+  display: 'flex',
+  alignItems: 'center',
+  gap: '1em',
+};
+
+export default function ProgressBar() {
+  const [value] = useState(Math.floor(Math.random() * 99));
+  return (
+    <div style={container}>
+      <div style={{ width: 100, height: 100 }}>
+        <CircularProgressbar value={value} />
+      </div>
+      <div>
+        <p style={{ fontSize: '2rem' }}>
+          {value}
+          %
+        </p>
+        <p style={{ fontSize: '1rem', color: '#a4a4a4' }}>
+          Completed
+        </p>
+      </div>
+    </div>
```

Comment:
## Optional
- [ ] I highly suggest that you add a space between these and consider working on responsiveness
<img width="370" alt="Screenshot 2023-08-24 at 18 08 04" src="https://github.com/katarighe/bookstore/assets/44745855/9dfd811f-7132-432e-8668-7bc235c7433d">